pypi-vuln/index-2021-07-27-combine-prs-workflow.rst

```diff
@@ -0,0 +1,101 @@
+=================================================
+Vulnerability in GitHub Actions workflow for PyPI
+=================================================
+
+An exploitable vulnerability in a GitHub Actions workflow for PyPI's source
+repository could allow an attacker to obtain write permissions against the
+``pypa/warehouse`` repository.
+
+* Disclosure date: **2020-07-25** (Reported via security policy on `pypi.org <https://pypi.org/security/>`_)
```

Comment:
I think this date should be `2021-07-27` (I reported it on `2021/07/27 3:19` UTC)